src/chigame/users/urls.py

```diff
@@ -2,9 +2,12 @@
 
 from chigame.users.views import user_detail_view, user_redirect_view, user_update_view
 
+from . import views
+
 app_name = "users"
 urlpatterns = [
     path("~redirect/", view=user_redirect_view, name="redirect"),
     path("~update/", view=user_update_view, name="update"),
     path("<int:pk>/", view=user_detail_view, name="detail"),
+    path('user-list/', views.user_list, name='user_list')
```

Comment:
here would be  path('user-list/', user_list, name='user_list')

Reply:
FIxed the format, but instead of path('user-list/', user_list, name='user_list'), I wrote path('user-list/', view=user_list, name='user_list') as I believe the 2nd format is the correct one.

Reply:
Yes, that is even better!!